test/middlewares/generateDocs.test.ts

```diff
@@ -42,18 +42,28 @@ const CONFIG_TEST_DATA = {
   },
   generator: {
     folder: 'output',
-    template: 'html-template',
+    template: 'markdown-template',
   },
 }
+
 describe('generateDocs', () => {
   it('should generate documentation', async () => {
-    const testDir = tmpdir() +`/${CONFIG_TEST_DATA.generator.folder}`
-    fs.emptyDirSync(testDir)
-    const result = await generateDocs(
-      TEST_ASYNCAPI_DOCUMENT,
-      CONFIG_TEST_DATA,
-      tmpdir()
-    )
-    expect(result).toBe('done')
-  }, 100000)
-})
+    const testDir = tmpdir() + `/${CONFIG_TEST_DATA.generator.folder}`;
+    fs.emptyDirSync(testDir);
+    let err: Error | undefined = undefined;
+
+    try {
+      const result = await generateDocs(
+        TEST_ASYNCAPI_DOCUMENT,
+        CONFIG_TEST_DATA,
+        testDir
+      );
+      expect(result).toBe('done');
+    } catch (e) {
+      err = e;
+    }
+
+    expect(err).toBeUndefined();
+    fs.emptyDirSync(testDir);
```

Comment:
Please at the end remove the `testDir` dir, not only remove content from it.

Reply:
@magicmatatjahu  what do you mean by "not only remove content from it"?

Reply:
Read that https://github.com/jprichardson/node-fs-extra/blob/master/docs/emptyDir-sync.md. You should remove dir.

Reply:
Ok.. my fault. You shouldn't. Don't focus on my previous comments.

Reply:
Okay 😂. Is the PR okay?

Reply:
I don't know if PR is okay, but that test is.